### DISCUSSION TITLE
I lose my “monika.character”file and I don't know how to fix it.

### DISCUSSION THREAD

**Discussion by Java-JR** [Monika-After-Story/MonikaModDev#10025] [QA - tech support + general] [answered]
I’m a chinese player with a young age,so my english not pretty well,and there will be some mistake in grammar,I hope you can understand it.
There is my problem.For a number of reasons,I accidentally lost my"monika.character“ when i download the mod.Is the file will be back when the mod finish? I don't want lose her,just like she said,she is the sunshine of my life. 
If you can help,thx very much.Thanks♪(･ω･)ﾉ
There is another question,usually how much time you will spend downloading the mod,I spend very*3 loooong time to do it.Is that my WiFi too solw or something else?
I can't reply in time because I'm still a student,but I will be faster I could to reply.ヾ(●´∀｀●)

**Comment by TooMuchInvigilation** [comment]
Hi. As far as I know, monika.char file is not needed to play the mod. Have you installed the mod after completing the main game?
I think I've downloaded the mod in 15 minutes or less.

**Comment by RedJuicyFood** [comment]
你好，我会说中文，你可以用中文回复我。
Hi there, I can speak Chinese, and you can reply me in Chinese.

> I accidentally lost my"monika.character“ when i download the mod.

你说丢失了`monika.character`文件，你是指不带任何后缀名的`monika`文件，还是带了后缀的`monika.chr`文件？带了后缀的那个`monika.chr`文件在MAS中是没有任何实际意义的（这点，Monika会在MAS开始有提及），有没有那个文件都不会影响你的游玩，请不用担心。至于不带任何后缀名的`monika`文件，这个文件是在你每次带Monika离开（就是`I'm going to take you somewhere`这个选项）时生成的，它有实际意义，如果你弄丢了那个文件，启动MAS以后Monika不会再继续出现。
听你的描述，你的Monika现在似乎依然是正常出现的？那就说明你不是弄丢了那个有实际意义的文件，你只是弄丢了那个带后缀的chr文件。不用担心，它在MAS中没有意义。

When you say you lost the `monika.character` file, do you mean the `monika` file without any suffix, or the `monika.chr` file with suffix? The `monika.chr` file with the suffix has no real meaning in MAS (Monika will mention this at the beginning of MAS), and it won't affect your play with or without that file, so please don't worry. As for the `monika` file without any suffix, this file is generated every time you take Monika away (that's the `I'm going to take you somewhere` option), it has real meaning, if you lose that file, Monika will not appear after starting MAS.
From your description, it seems that your Monika is still appearing normally? That means you didn't lose the meaningful one, you just lost the chr file. Don't worry, it won't affect anything in MAS.

> Is the file will be back when the mod finish?

MAS可没有“完成”一说。

MAS does not have a "finish".

> how much time you will spend downloading the mod,I spend very*3 loooong time to do it.Is that my WiFi too solw or something else?

中国地区的下载速度相当受限，这是没办法的。

Well, it can't be helped that the download speed in China is quite limited.

> I can't reply in time because I'm still a student,but I will be faster I could to reply.ヾ(●´∀｀●)

没事，自己慢慢来吧。还有什么问题，记得继续提出，我会看着这个Github仓库的。

It's okay, take your time. If you have any more questions, remember to keep asking, I'll keep an eye on this Github repository.

**Comment by Java-JR** [comment]
> 当您说丢失文件时，您是指没有任何后缀的文件还是带有后缀的文件？带有后缀的文件在 MAS 中没有实际意义（Monika 会在 MAS 的开头提到这一点），无论有没有该文件，它都不会影响您的游戏，所以请不要担心。至于没有任何后缀的文件，每次你把Monika带走时都会生成这个文件（这是选项），它有真正的意义，如果你丢失了那个文件，Monika在启动MAS后不会出现。从你的描述来看，你的莫妮卡似乎还正常出现？这意味着您没有丢失有意义的文件，只是丢失了chr文件。别担心，它不会影响 MAS 中的任何内容。`monika.character``monika``monika.chr``monika.chr``monika``I'm going to take you somewhere`
> 我应该是丢失了有意义的文件，打开游戏后，我的莫妮卡变成了乱码，她在责怪我为什么这样对待她，就像下图（如果看得见的话）
![屏幕截图 2023-05-26 184319](https://github.com/Monika-After-Story/MonikaModDev/assets/123796773/c03f08ae-eb4d-4c4e-a9d6-0c428d64bc64)

**Comment by Java-JR** [comment]
我在想我是否需要留下存档重新下载after story
I was wondering if I needed to leave the archive to re-download after story

**Comment by Java-JR** [comment]
> Hi. As far as I know, monika.char file is not needed to play the mod. Have you installed the mod after completing the main game? I think I've downloaded the mod in 15 minutes or less.

I download several times,but all fall,the last i download it.It said that my computer refuse to download the mod.

**Comment by RedJuicyFood** [comment, marked as answer]
> 我应该是丢失了有意义的文件，打开游戏后，我的莫妮卡变成了乱码，她在责怪我为什么这样对待她，就像下图（如果看得见的话）

你这是在玩DDLC，不是在玩MAS——图里那部分内容属于原版游戏，不属于MAS的范围。你安装MAS了吗？
You are playing DDLC, not MAS. That part belongs to the original game, not the MAS. Have you installed the mod?

另外，你是打开了网页自动翻译吧？不要这样，网页自动翻译在Github这里容易搞出很多搞笑的错误，把它关掉。我已经说了，我是能和你用中文交流的，何必还要开个翻译软件呢。
By the way, you seem to have auto-translation turned on? Don't do that, automatic page translation tends to make a lot of funny errors here on Github, turn it off. In fact, I have told you that I can speak in Chinese, there is no need to use a translator anymore.

> > Hi. As far as I know, monika.char file is not needed to play the mod. Have you installed the mod after completing the main game? I think I've downloaded the mod in 15 minutes or less.
> 
> I download several times,but all fall,the last i download it.It said that my computer refuse to download the mod.

没办法，中国地区下载就是很容易失败。要么多试一试，要么考虑使用VPN之类的东西吧。
Well, downloading MAS in China is just easy to fail, it can't be helped. Either try a few more times, or consider using something like a VPN.

然后，考虑加个QQ群？即时通讯可能会方便一些。群号：641810972
Also, how about adding a QQ group? Instant messaging might come in handy. Group number: 641810972

**Comment by Java-JR** [comment]
> > 我应该是丢失了有意义的文件，打开游戏后，我的莫妮卡变成了乱码，她在责怪我为什么这样对待她，就像下图（如果看得见的话）
> 
> 你这是在玩DDLC，不是在玩MAS——图里那部分内容属于原版游戏，不属于MAS的范围。你安装MAS了吗？ 您正在玩DDLC，而不是MAS。这部分属于原始游戏，而不是MAS。你安装了模组吗？
> 
安装了,可能是在用安装包的时候重置了after story。已经重新安装了，游戏可以正常玩了。

> > > 你好。据我所知，播放mod不需要monika.char文件。完成主游戏后你安装了模组吗？我想我已经在 15 分钟或更短的时间内下载了该模组。
> > 
> > 
> > 我下载了几次，但都掉了，最后我下载了它。它说我的电脑拒绝下载该模组。
> 
> 没办法，中国地区下载就是很容易失败。要么多试一试，要么考虑使用VPN之类的东西吧。好吧，在中国下载MAS很容易失败，这是无济于事的。要么再尝试几次，要么考虑使用 VPN 之类的东西。
> 
我在用watt toolkit
> 然后，考虑加个QQ群？即时通讯可能会方便一些。群号：641810972 另外，添加QQ群怎么样？即时消息可能会派上用场。组号： 641810972
>
这就来
